.github/workflows/release.yaml

```
@@ -351,3 +362,20 @@ jobs:
               tmp/"${channel}"/* \
               '${{ env.itch_page }}':"${channel#package-}"
           done
+
+  # Upload to GitHub Pages.
+  deploy-to-pages:
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      # Required to upload to GitHub Pages.
+      pages: write
+      # Required to verify the upload originates from an appropriate source.
+      id-token: write
+    environment:
+      name: github-pages
+      url: ${{ steps.pages.outputs.page_url }}
+
+    steps:
+      - name: Deploy to GitHub Pages
+        uses: actions/deploy-pages@v4
```

```suggestion
      url: ${{ steps.deploy.outputs.page_url }}

    steps:
      - name: Deploy to GitHub Pages
        id: deploy
        uses: actions/deploy-pages@v4
```
The `${{ steps.*.outputs.page_url }}` actually comes from _this_ step, so this step needs the ID, sorry about that 😅